group_vars/ironic_all.yml

```diff
@@ -14,3 +14,14 @@
 # limitations under the License.
 
 lxc_container_variant: "ironic-{{ rpc_release }}"
+ironic_osa_config:
```

Comment:
Ran a test with this change and got this error:
```
fatal: [localhost]: FAILED! => {"failed": true, "msg": "the field 'args' has an invalid value, which appears to include a variable that is undefined. The error was: 'ironic_osa_config' is undefined\n\nThe error appears to have been in '/opt/rpc-openstack/scripts/ironic/playbooks/ironic-pre-config.yml': line 63, column 7, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n    - name: Configure ironic OSA variables\n      ^ here\n"}
```
The playbooks that use these variables run on localhost and utility_all[0], so these variables aren't defined for those hosts.